server/sonar-server/src/test/java/org/sonar/server/qualityprofile/QProfileFactoryMediumTest.java

```diff
@@ -208,7 +220,7 @@ public void fail_renaming_if_name_already_exists() {
   @Test
   public void delete() {
     initRules();
-    db.qualityProfileDao().insert(dbSession, QProfileTesting.newXooP1("org-123"));
+    db.qualityProfileDao().insert(dbSession, QProfileTesting.newXooP1(organization.getUuid()));
```

Comment:
If I had reviewed the changes on `QProfileTesting` I would have advised to use `OrganizationDto` arguments instead of `String` ones: in practice, you almost always have a `OrganizationDto` at hand in UTs and you get a strongly typed parameter